test/contrib/isotp.uts

```diff
@@ -865,10 +865,8 @@ assert(succ.is_set())
 with ISOTPSocket(MockCANSocket(), sid=0x641, did=0x241) as s:
     assert(s.impl.rx_thread.is_alive())
     s.close()
-    s.impl.rx_thread.join(5)
+    assert(not s.impl.rx_thread.join())
```

Comment:
Could you leave a timeout in here ?

Reply:
I didn't remember making this change, maybe it was a failure in the merge.